.github/workflows/ci.yml

```
@@ -25,20 +25,20 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
     with:
       envs: |
-        - linux: test-oldestdeps-cov-xdist
+        - linux: test-py39-oldestdeps-cov
           python-version: 3.9
-        - linux: test-xdist
+        - linux: test-py39
           python-version: 3.9
-        - linux: test-xdist
+        - linux: test-py310
           python-version: 3.10
-        - linux: test-xdist
+        - linux: test-py311
           python-version: 3.11
-        - macos: test-xdist
+        - macos: test-py311
           python-version: 3.11
-        - linux: test-cov-xdist
+        - linux: test-py3x-cov
```

```suggestion
        - linux: test-cov
```
The `cov` distinguishes it sufficiently.